### ISSUE TITLE
Inserts two spaces (would prefer one) after timestamp

### ISSUE THREAD

**Issue by alexpriest** [QWxleA/logseq-interstitial-heading-plugin#8]
Hiya! The plugin seems to insert two spaces after the timestamp—generally I would just prefer one. Seems to do the same two spaces with a custom time stamp, too.

Thanks!

**Comment by willettjf** [comment]
Yes, would be great to insert only one space so you don't have to backspace before typing.

**Comment by jaroet** [comment]
I too run in the 2 spaces after the time. I would even go one further and would like to see the fields prefix and suffix. I personally like to have a -> behind the time.
